src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -66,42 +65,48 @@ public StandardSlackService(String baseUrl, String teamDomain, String token, Str
      */
     @Deprecated
     public StandardSlackService(String baseUrl, String teamDomain, String token, String authTokenCredentialId, boolean botUser, String roomId, boolean replyBroadcast) {
-        this(baseUrl, teamDomain, botUser, roomId, replyBroadcast);
+        this(baseUrl, teamDomain, botUser, roomId, replyBroadcast, authTokenCredentialId);
         this.populatedToken = getTokenToUse(authTokenCredentialId, token);
         if (this.populatedToken == null) {
             throw new IllegalArgumentException("No slack token found, setup a secret text credential and configure it to be used");
         }
     }
 
-    /**
-     * @param baseUrl        the full url to use, this is an alternative to specifying teamDomain
-     * @param teamDomain     the teamDomain inside slack.com to use
-     * @param botUser
-     * @param roomId         a semicolon separated list of rooms to notify
-     * @param replyBroadcast
-     * @param populatedToken a non-null token to use for authentication
-     */
+    @Deprecated
     public StandardSlackService(String baseUrl, String teamDomain, boolean botUser, String roomId, boolean replyBroadcast, String populatedToken) {
-        this(baseUrl, teamDomain, botUser, roomId, replyBroadcast);
+        this(new StandardSlackServiceBuilder()
```

Comment:
sorry I missed this before
```suggestion
        this(builder()
```